From d5656ffca80a3261db3323db19358762bf10a88e Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Sat, 1 Sep 2018 14:32:52 +0200
Subject: [PATCH] systemd-udev.service remove MountFlags

Remove MountFlags=slave. The default MountFlags=shared is now active.
MountFlags=shared ensures that mounts and umounts done by udev rules are
propagated to host. See also

http://www.volkerschatz.com/unix/advmount.html
http://www.freedesktop.org/software/systemd/man/systemd.exec.html

With systemd version 239 the flag PrivateMounts=yes must be removed to
get the old behavior.

Signed-off-by: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
---
 units/systemd-udevd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/units/systemd-udevd.service.in b/units/systemd-udevd.service.in
index 6a3814e5d9..e3f3ad3293 100644
--- a/units/systemd-udevd.service.in
+++ b/units/systemd-udevd.service.in
@@ -25,7 +25,6 @@ ExecStart=@rootlibexecdir@/systemd-udevd
 KillMode=mixed
 WatchdogSec=3min
 TasksMax=infinity
-PrivateMounts=yes
 MemoryDenyWriteExecute=yes
 RestrictRealtime=yes
 RestrictAddressFamilies=AF_UNIX AF_NETLINK AF_INET AF_INET6
-- 
2.16.4

